Packages/InputSystem/Runtime/CallbackContextEvent.cs

```diff
@@ -0,0 +1,74 @@
+using UnityAtoms;
+using UnityEngine.InputSystem;
+
+public class CallbackContextEvent<T, P, C, V, E1, E2, F, VI> : BaseAtom
+    where T : struct
+    where P : struct, IPair<T>
+    where C : AtomBaseVariable<T>
+    where V : AtomVariable<T, P, E1, E2, F>
+    where E1 : AtomEvent<T>
+    where E2 : AtomEvent<P>
+    where F : AtomFunction<T, T>
+    where VI : AtomVariableInstancer<V, P, T, E1, E2, F>
+{
+    public E1 any;
```

Comment:
Make variable private and use the SerializeField attribute. Change the name to have a leading undescore. Eg. `_any`.